components/FlyOut.vue

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+import SearchTrackers from "./SearchTrackers.vue";
+
+export default defineComponent({
+    components: { SearchTrackers },
+    props: {
+        initOpen: {
+            type: Boolean,
+            default: false
+        }
+    },
+    data () : Object {
+        return {
+            open: this.initOpen
+        };
+    }
+});
+</script>
+
+<template>
+  <div id="flyout-wrapper">
+    <b-button id="flyout-toggle" v-b-toggle.flyout class="shadow-none" squared @click="open= ! open">
+      <img v-if="!open" id="menu-closed" src="/svg/burger-menu.svg" width="32" height="32">
+      <img v-else id="menu-opened" src="/svg/close-menu.svg" width="32" height="32">
+    </b-button>
+    <b-sidebar id="flyout" bg-variant="dark" no-header shadow @hidden="open=false">
+      <!-- because of no-header, we need an additional horizontal break-->
+      <hr class="flyout-top">
+      <SearchTrackers />
+    </b-sidebar>
+  </div>
+</template>
+
+<style lang="scss">
+  $dark-blue: #141f2e;
+
+  #flyout {
+    height: 80%;
+    width: 15%;
+    margin-top: 5%;
+    border-radius: 0 20px 20px 0;
+    position: fixed;
+    z-index: -1;
+    background-color: $dark-blue !important;
```

Comment:
I have some issues with using scoped, by it overwriting bootstrap. I don't know if it is needed to fix the !important tag for this PR, or if this can be seen as a part of next week's goal of making everything more responsive.